### DISCUSSION TITLE
Navidrome and Snapcast

### DISCUSSION THREAD

**Discussion by lambchop01** [navidrome/navidrome#2800] [Show and tell]
I have managed to get navidrome to work with snapcast in jukebox mode for whole home audio in docker.  I ended up recreating the docker container with mpd and alsa installed as well, so I could pipe the audio from alsa to /tmp/snapfifo which the snapserver will play.

If anyone is struggling to do something simillar you are welcome to take a look at [my set up](https://github.com/lambchop01/navidrome-snapcast).

**Comment by tcurdt** [comment]
I got this also working but I am having problem with clients.
Have you found clients that work well enough?

https://github.com/navidrome/navidrome/issues/3022#issuecomment-2647771483
